### PR TITLE
feat: stamp issuer utilization rate test

### PR DESCRIPTION
**Pull request by istae**
### Checklist

- [ ] I have read the [coding guide](https://github.com/ethersphere/bee/blob/master/CODING.md).
- [ ] My change requires a documentation update, and I have done it.
- [ ] I have added tests to cover my changes.
- [ ] I have filled out the description and linked the related issues.

### Description
Minimum depth is now 22.

### CODE REVIEW

File: pkg/postage/stampissuer_test.go [hunk 1]
```diff
@@ -203,6 +206,38 @@ func Test_StampIssuer_inc(t *testing.T) {
 	})
 }
 
+func TestUtilization(t *testing.T) {
+	// t.Skip("meant to be run for ad hoc testing")
```
Comment (mrekucci) [inline pkg/postage/stampissuer_test.go:210]: You're missing `t.Parallel()`.

File: pkg/postage/stampissuer_test.go [hunk 2]
```diff
@@ -203,6 +206,38 @@ func Test_StampIssuer_inc(t *testing.T) {
...
+
+	for depth := uint8(17); depth < 25; depth++ {
+		sti := postage.NewStampIssuer("label", "keyID", make([]byte, 32), big.NewInt(3), depth, postage.BucketDepth, 0, true)
+
+		var count uint64
+
+		eg := errgroup.Group{}
```
Comment (mrekucci) [inline pkg/postage/stampissuer_test.go:217]: nitpick: `var eg errgroup.Group` will do.

File: pkg/postage/stampissuer_test.go [hunk 3]
```diff
@@ -203,6 +206,38 @@ func Test_StampIssuer_inc(t *testing.T) {
...
+						return err
+					}
+					atomic.AddUint64(&count, 1)
+				}
+			})
+		}
+
+		err := eg.Wait()
+		if !errors.Is(err, postage.ErrBucketFull) {
+			t.Fatal("err should be bucket full")
```
Comment (mrekucci) [inline pkg/postage/stampissuer_test.go:233]: `t.Fatalf("want: %v; have: %v", postage.ErrBucketFull, err)`

**Review by mrekucci — APPROVED** [review]

**Comment by istae** [comment]
```
go test  -run ^TestUtilization$ github.com/ethersphere/bee/pkg/postage -count=1 -v
=== RUN   TestUtilization
    stampissuer_test.go:236: depth: 17, actual utilization: 0.037651
    stampissuer_test.go:236: depth: 18, actual utilization: 0.102993
    stampissuer_test.go:236: depth: 19, actual utilization: 0.223402
    stampissuer_test.go:236: depth: 20, actual utilization: 0.349894
    stampissuer_test.go:236: depth: 21, actual utilization: 0.491061
    stampissuer_test.go:236: depth: 22, actual utilization: 0.610335
    stampissuer_test.go:236: depth: 23, actual utilization: 0.672418
    stampissuer_test.go:236: depth: 24, actual utilization: 0.757001
```

**Review by mrekucci — APPROVED** [review]

**Pull request merged**